style.css

```diff
@@ -0,0 +1,74 @@
+/* mobile first */
+
+/* min-width: 400px */
+
+html {
+  font-family: "Inter", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+body {
+  width: 375px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 1em;
+  margin-top: 3em;
+  padding-left: 3.5em;
+  padding-right: 3.5em;
+}
+
+h2 {
+  font-size: 3.4em;
+  font-weight: 700;
+  line-height: 1.3em;
+  margin: 0.2em;
+  text-align: center;
+}
+
+p {
+  font-size: 1.7em;
+  font-weight: 400;
+  line-height: 28px;
+  text-align: center;
+  width: 18em;
+}
+
+/* Classes */
+
+.primary_color {
+  color: #1e488f;
+}
+
+.secondary_color {
+  color: #172b4d;
+}
```

Comment:
- [ ] Kindly, I would recommend using that your section uses the font and colors (background and font) defined for the template :joy: 

    Remember it is important to try to replicate the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) as closely as possible to give confidence to recruiters in your ability to produce exact business requirements :pray:

**Your design** | **Figma design**
| --- |--- |
![image](https://user-images.githubusercontent.com/102757126/205109388-c25136e9-57ed-45ce-9867-41937badd1cc.png) | ![image](https://user-images.githubusercontent.com/102757126/205109437-a7f8c5b3-4ebe-4798-aaaa-16f70d89fe9c.png)

Reply:
All the observations mentioned in the above comments have been implemented.
Thank you